app/helpers/host_helper.rb

```diff
@@ -1,4 +1,11 @@
 module HostHelper
   include_concern 'TextualSummary'
   include_concern 'ComplianceSummaryHelper'
+
+  def get_host_info
+    devices = get_devices
+    os_info = get_os_info
+    vmm_info = get_vmm_info
+    [devices, os_info, vmm_info]
```

Comment:
yea I've created the PR just as a raw PR (or WIP) .. my point is to see review about the moving the code in to the helpers .. there are many things to improve in the code, but surely the code that makes your eyes bleeding, write it, I'll fix that ;-)